### ISSUE TITLE
[16.0][mass_editing] Action "Mass Edit" doesn't appear in "Server Actions" or in Actions during objects choosing

### ISSUE THREAD

**Issue by Querens** [OCA/server-ux#721]
## Module
mass_editing
## Describe the bug
Action "Mass Edit" doesn't appear in "Server Actions"

## To Reproduce
**Affected versions**:16.0

1. Download the archive https://apps.odoo.com/apps/modules/16.0/server_action_mass_edit/
2. Unzip it to modules folder
3. Update the list of apps in Odoo, activate the module
4. Go to Settings / Technical / Actions / Server Actions

**Expected behavior**
The action "Mass Edit" to be present

**Additional context**
At first I thought that I didn't understand the configuration proccess right( which still can be the case) and tried to create an action manually by adding some new action. In particular, I wanted to be able to change teams for tickets in the helpdesk module. So I've created an action, added the module "Helpdesk Ticket", chose "Action To Do: Mass Edit Records" and added the field "Team (helpdesk.ticket)". It didn't help, so I went back to Actions and also did "CREATE CONTEXTUAL ACTION"( frankly I have no idea what it does, but it didn't help either). The way I've been checking whether it works is by going to helpdesk -> tickets, choosing some random tickets and searching for this newly created action in Actions.

**Comment by Querens** [comment]
Seems like I've messed up the model I chose to mass edit. I've found my "Mass Edit" action right inside tickets, but I need it in the list of all tickets.

**Comment by Querens** [comment]
Very strange. I did nothing but "Mass edit" appeared on the list of all tickets and now I can select any number of tickets and apply "Mass edit" to them. In "Mass edit" there is a "team" selection as expected, but I don't understand where those came from because those are unrelated to teams I have. Available options are "don't touch", "set" and "remove". I have no idea what are those, when I tried to apply something the team just become "None".
